src/button.js

```diff
@@ -29,7 +29,9 @@ function Button (command, opts) {
     this._disabled = false;
     this._label = opts.label || '';
     this._errback = opts.errback;
-    this._insightsVerb = opts.insightsVerb || opts.label || '';
+    this._insightsVerb = opts.insightsVerb === null ?
+        null :
+        opts.insightsVerb || opts.label || null;
```

Comment:
Bleh, it's not that easy if we want to allow buttons to not fire events. Oh well.